### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by prakashchoudhary07**
### Issue

#561 

stepper.js test was previously skipped, Track this in a separate issues

### Is Development Tested?

- [x] Yes
- [ ] No

### CODE REVIEW

File: app/constants/urls.js [hunk 1]
```diff
@@ -1,24 +1,58 @@
...
 
-export const APPS = {
-  HOME: ENV.APPS?.HOME,
-  WELCOME: ENV.APPS?.WELCOME,
-  EVENTS: ENV.APPS?.EVENTS,
-  MEMBERS: ENV.APPS?.MEMBERS,
-  STATUS: ENV.APPS?.STATUS,
-  PROFILE: ENV.APPS?.PROFILE,
-  LIVE: 'live',
+const SCHEMA = 'https://';
```
Comment (satyam73) [inline app/constants/urls.js:3]: Isn't the name `SCHEMA` seems confusing?
Can we name it `PROTOCOL`?

**Review by satyam73 — CHANGES_REQUESTED** [review]

File: app/constants/urls.js (hunk 1 above)
Comment (prakashchoudhary07) [reply app/constants/urls.js]: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#scheme

Comment (prakashchoudhary07) [reply app/constants/urls.js]: I can change it its confusing

**Review by satyam73 — COMMENTED** [review]
There is conflicts in the PR, please sync with the current branch.

File: app/constants/urls.js (hunk 1 above)
Comment (satyam73) [reply app/constants/urls.js]: Hmm got it👍, we can keep it

**Review by satyam73 — DISMISSED** [review]
LGTM

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**